libcxx/test/std/numerics/c.math/hermite.pass.cpp

```diff
@@ -0,0 +1,310 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14
+
+#include <cassert>
+#include <cmath>
+#include <limits>
+#include <vector>
+#include <array>
+
+#include <assert_macros.h>
+
+namespace {
```

Comment:
Generally, we don't have anonymous namespaces in the tests. It's not like we get much of a benefit from it.

Reply:
Fixed in 71dcd12a17d321d46af15f13e7f9d3ee02604940.